### PR TITLE
Adding Fastly TLS resources to CICD

### PR DESCRIPTION
**Pull request by rajdnp**
Description : 
1.Adding below Resources to CICD : 
  Fastly-Tls-Certificate
  Fastly-Tls-Domain
  Fastly-Tls-PrivateKeys
2.Fixing Prod pipeline issue for existing resources 

Testing: 
Local Alpha, Beta and Prod are successful 

image

### CODE REVIEW

File: deregister-all.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 # Does not un-publish any published versions.
 #
-# Run this from the resource directory, for example `GitHub-Repositories-Repository/`
+# Run this from the resource directory, for example `Databricks-Clusters-Job/`
```
Comment (joshuadeanhall) [inline deregister-all.sh:7]: Can we make this say fastly instead?

**Review by joshuadeanhall — CHANGES_REQUESTED** [review]

File: Fastly-Tls-Certificate/setup.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# This script runs in the buildspec before testing
+#
+# It relies on environment variables to be set to replace placeholder
+# values in `example-inputs`.
+#
+#   FASTLY_SERVICE_ID
+#   FASTLY_VERSION_ID
+#   Example:
+#
+#    GITHUB_ORG=ericzbeard-aws-cep-testing GITHUB_REPO=test-stable-1 AWS_PROFILE=ezbeard-cep GITHUB_SHA=6907b5086cfdc46381af05d94c2e41ac6812a962 ./setup.sh
```
Comment (ericzbeard) [inline Fastly-Tls-Certificate/setup.sh:12]: We should delete this line

File: Fastly-Tls-Domain/setup.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# This script runs in the buildspec before testing
+#
+# It relies on environment variables to be set to replace placeholder
+# values in `example-inputs`.
+#
+#   FASTLY_SERVICE_ID
+#   FASTLY_VERSION_ID
+#   Example:
+#
+#    GITHUB_ORG=ericzbeard-aws-cep-testing GITHUB_REPO=test-stable-1 AWS_PROFILE=ezbeard-cep GITHUB_SHA=6907b5086cfdc46381af05d94c2e41ac6812a962 ./setup.sh
```
Comment (ericzbeard) [inline Fastly-Tls-Domain/setup.sh:12]: Delete

File: Fastly-Tls-PrivateKeys/setup.sh [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+# This script runs in the buildspec before testing
+#
+# It relies on environment variables to be set to replace placeholder
+# values in `example-inputs`.
+#
+#   FASTLY_SERVICE_ID
+#   FASTLY_VERSION_ID
+#   Example:
+#
+#    GITHUB_ORG=ericzbeard-aws-cep-testing GITHUB_REPO=test-stable-1 AWS_PROFILE=ezbeard-cep GITHUB_SHA=6907b5086cfdc46381af05d94c2e41ac6812a962 ./setup.sh
```
Comment (ericzbeard) [inline Fastly-Tls-PrivateKeys/setup.sh:12]: Delete

File: publish-resource-regions.sh [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+#
+# Run this from the resource folder
+
+regions_to_publish=(us-east-1 af-south-1 ap-east-1 ap-northeast-1 ap-northeast-2 ap-northeast-3 ap-south-1 ap-southeast-1 ap-southeast-2 ap-southeast-3 ca-central-1 eu-central-1 eu-north-1 eu-south-1 eu-west-1 eu-west-2 eu-west-3 me-central-1 me-south-1 sa-east-1 us-east-2 us-west-1 us-west-2 eu-central-2 eu-south-2 ap-south-2)
+
+# Errors:
+
+# This is a registry bug
+
+#About to poll test status for arn:aws:cloudformation:eu-south-2:387586997764:type/resource/AwsCommunity-S3-DeleteBucketContents/00000001
```
Comment (ericzbeard) [inline publish-resource-regions.sh:13]: Remove these comments

**Review by ericzbeard — CHANGES_REQUESTED** [review]

**Review by ericzbeard — APPROVED** [review]

**Pull request merged**